packages/client/src/components/pages/editQuestionPage/EditQuestionPage.tsx

```diff
@@ -228,11 +229,24 @@ const EditQuestionPage: React.FC<IEditQuestionPageProps> = ({
         <div className={classes.btns}>
           <Button onClick={() => setShowMsg(true)}>« Back</Button>
           <div className={classes.filler} />
-          <Button>Show</Button>
+          <Button
+            onClick={() => setShowId(showId === question.id ? "" : question.id)}
+          >
+            {showId === question.id ? "Shrink" : "Show"}
+          </Button>
           <Button success onClick={submitForm}>
             Save »
           </Button>
         </div>
+        <div>
+          {showId === question.id && (
+            <div key={question.id + 5}>
+              <div style={{ textAlign: "start" }}>
+                <DisplayQuestion question={question} />
+              </div>
+            </div>
+          )}
+        </div>
```

Comment:
```suggestion
     {show && (            
              <div style={{ textAlign: "start" }}>
                <DisplayQuestion question={question} />
              </div>
          )}
          )}
        </div>
```
why do you need "key"?